includes/data.php

```diff
@@ -1649,7 +1649,7 @@ function pods_serial_comma( $value, $field = null, $fields = null, $and = null,
 		'fields'      => $fields,
 		'and'         => $and,
 		'field_index' => $field_index,
-		'separator'   => ',',
+		'separator'   => ', ',
```

Comment:
```suggestion
		'separator'   => ',',
```

We can't add the space here because they could pass it in without the space and that would break some compatibility